paddle/go/optimizer/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
`repo/go` is mostly for go project and their bindings. Maybe optimizer is better located under `repo/paddle/pserver`?